### DISCUSSION TITLE
What if my university domain isn't recognised?

### DISCUSSION THREAD

**Discussion by EY-glowglowglow** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#37] [2.0 Getting your free Azure credits]
If you are having issues assessing Azure vouchers because your university domain is not recognised, please comment your student email/university domain below.

**Comment by AnishDixit** [comment]
University domain not recognized, please help.

Name: Anish Dixit
Email ID: anish.dixit@spit.ac.in

**Reply by EY-glowglowglow** [reply]
Your domain has been added to the database. Please try requesting for a voucher again.

**Comment by thitiwutwin** [comment]
University domain is not recognized

University : Mahidol
Domain : student.mahidol.ac.th
Email : thitiwut.mar@student.mahidol.ac.th

**Reply by anprieto** [reply]
Your domain has been added to the database.  Please try again and let us know if you still experience problems.

**Reply by thitiwutwin** [reply]
Thank you, but it still doesn't available.

" You're not eligible for an Azure free account "

**Reply by anprieto** [reply]
ok, sorry, I misread your message and thought you were trying to get your free voucher.
If this problem is on Azure, my colleagues will help you, probably on Monday.

**Reply by EY-glowglowglow** [reply]
Hi there, to be eligible, participants must be students enrolled at a university in one of [140 countries](https://azure.microsoft.com/en-us/free/free-account-students-faq/). 
If you are ineligible, you can try one of two alternative offers recommended by Microsoft in the FAQ - Azure for Students Starter (no credit card required) and [Azure free account](https://azure.microsoft.com/en-us/free/) (credit card required).

**Reply by thitiwutwin** [reply]
Thank you.

**Comment by Kishen35** [comment]
Hi, my university domain is not recognized

Name: Kishen Shanthan A/L Jegathes
University: Asia Pacific University of Technology & Innovation (APU)
Country: Malaysia
Domain: mail.apu.edu.my
Email: tp055296@mail.apu.edu.my

**Reply by EY-glowglowglow** [reply]
Hi there, I've added mail.apu.edu.my to the database. Please try requesting for an Azure voucher again. :)

**Comment by Ziffer-byakuya** [comment]
Hi, my university domain is not recognized

Name: Zhaowei Han
University: The Chinese University of Hong Kong, Shenzhen
Country: China
Domain: link.cuhk.edu.cn
Email: 119010092@link.cuhk.edu.cn

**Reply by EY-glowglowglow** [reply]
Hi there,

Azure for Student vouchers are not currently available for students in China. The alternatives are:  Azure for Students Starter (no credit card required) and Azure free account (credit card required).

Hope that helps!
